datamade/django-councilmatic-notifications
# Adapt the app for Councilmatic 1.0

## Overview

Make sure the app is compatible with `django-councilmatic` v1.0. The major changes in this update include:

- Support Django 2.0+
- Refactor queries in `send_notifications` management command to use the ORM and the new OCD model structure
- Write some basic tests for the app

## Testing instructions

- Make a virtualenv for the project and install it for testing with `pip install -e .[tests]`
- Run `pytest` and confirm all tests pass
